ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java

```diff
@@ -53,14 +54,16 @@ public void addIcon(String icon) {
         driver.findElement(EXTENSION_FORM_ICON).sendKeys(icon);
     }
 
-    public boolean addPresent() {
-        try {
-            return driver.findElement(ADD) != null;
-        } catch (Exception ex) {
-            return false;
-        }
+    public Boolean addPresent() {
+
+			try {
+				return driver.findElement(ADD) != null;
+			} catch (Exception ex) {
```

Comment:
this is doing no change at all but rather miss-formatting, you need to revert this part

Reply:
Resolved thanks